### PR TITLE
Essay and Demo proposal

### PR DESCRIPTION
**Pull request by protopapa**

### CODE REVIEW

**Comment by protopapa** [comment]
The commits from presentation and open task are shown. But its because I did not squash the commits. I can do that later. This pull request is only for essay and demo.

**Comment by protopapa** [comment]
Fixed.

File: contributions/demo/protopapa-parumita/README.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+Members:
+
+Parumita Saha parumita@kth.se
+Anastasia Protopapa anapro@kth.se
```
Comment (gluckzhang) [inline contributions/demo/protopapa-parumita/README.md]: Here your email address is `anapro`, while in the folder's name, your part is `protopapa`, please rename it, thanks :)

File: contributions/essay/protopapa-parumita/README.md [hunk 2]
```diff
@@ -0,0 +1,3 @@
+Authors: Parumita Saha parumita@kth.se , Anastasia Protopapa anapro@kth.se Topic: Cloud Workspaces
```
Comment (gluckzhang) [inline contributions/essay/protopapa-parumita/README.md]: Same issue here, use your email address to name the folder.

**Pull request merged**